core/templates/components/rubrics-editor/rubrics-editor.component.html

```diff
@@ -87,6 +99,20 @@
     padding: 10px;
   }
 
+  .rubric-error-message {
+    color: red;
```

Comment:
Please prefer hex codes for colours. See style guide [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).

Reply:
Done